src/main.rs

```diff
@@ -380,5 +440,13 @@ fn main() -> Result<(), String> {
         .unwrap();
     file.write_all(data.as_bytes()).unwrap();
 
+    // Write how long the entire operation took
+    // This might go at board level instead, not sure
+    let time_filename = (&opt.resultspath).to_owned() + "/time.txt";
+    let mut time_file = File::create(time_filename).unwrap();
+    let total_duration = end.duration_since(start);
+    let duration_str = format!("Elapsed: {:?}", total_duration);
+    time_file.write_all(duration_str.as_bytes()).unwrap();
+
```

Comment:
I think this should also be behind some CLI flag. For the case of analyzing multiple functions at the same time it is not as useful